sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -338,6 +338,10 @@ class EventDataBatch(object):
     **Please use the create_batch method of EventHubProducerClient
     to create an EventDataBatch object instead of instantiating an EventDataBatch object directly.**
 
+    **It is highly recommended not to update the value of the instance variable `max_size_in_bytes` on an
+    instantiated EventDataBatch object which might lead to sending failure as the Event Hub service
+    has a fixed quota on the maximum allowed size of events.**
```

Comment:
maybe something like this?
```suggestion
    **WARNING: Updating the value of the instance variable `max_size_in_bytes` on an instantiated EventDataBatch object is HIGHLY DISCOURAGED. The updated `max_size_in_bytes` value may conflict with the maximum size of events allowed by the Event Hubs service and result in a sending failure.**
```

Reply:
your wording is far more better than mine, I'll take yours :P
thanks!